### PR TITLE
Fix to avoid KeyError at /sso/saml/

### PR DESCRIPTION
**Pull request by santiagovassallo**
Added if to control the existence of the key in dict saml_data inside. In this way, if there are fields that are not returned by the server the application still works (provided that the inexistent keys have a default in the model)

### CODE REVIEW

**Comment by defionscode** [comment]
So, this changes behavior. I think I want to see this as a configurable boolean option with the default behavior being the current one which errors if there is no match. Something like `strict_mapping` as a boolean or whatever you want to call it, maybe even `default_mapping_to_user_model` or whatever. What do you tink?

**Comment by santiagovassallo** [comment]
I liked the idea! I'll add a setting variable called `SAML_USERS_STRICT_MAPPING` default to `True` to keep the original behavior.

**Comment by defionscode** [comment]
Excellent! And please do update the README as well if you could. Thanks!

File: README.md [hunk 1]
```diff
@@ -221,6 +221,18 @@ Defaults to False
 SAML_USERS_SYNC_ATTRIBUTES = True
 ```
 
+
+**SAML_USERS_STRICT_MAPPING (optional):**
+Specifies if every user attribute defined in SAML_USER_MAP must be present
+in the saml response or not. If set to False, make sure the user model have
+ the atributes that may not be present in the response with null=True or a default value.
+
```
Comment (defionscode) [inline README.md]: slight wording style change proposal
`If set to False, ensure your Django user model defines those attributes which your IdP doesn't expose in the SAML response. You could do this by setting `null=True` or a default value in your user model.`

**Pull request merged**